docs/restore.md

```diff
@@ -46,3 +46,32 @@ INFO[2022/10/31 06:34:50] Done restoring
 ```
 
 Continue with procedure from [Rancher documentation](https://docs.ranchermanager.rancher.io/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/migrate-rancher-to-new-cluster)
+
+:::warning warning
+Try to not restart `rancher-system-agent` on elemental nodes, otherwise it can stuck in following error
+
+`panic: error while connecting to Kubernetes cluster: the server has asked for the client to provide credentials`
+
+If this problem occurs please go with procedure below.
```

Comment:
I wonder if we should just start a troubleshooting page and start putting this info in a converged place.